### PR TITLE
npm audit fix

### PR DESCRIPTION
**Pull request by jfrerich**
Update results from npm audit fix

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "eslint-plugin-react": "7.10.0",
     "jest": "24.8.0",
     "lodash": ">=4.17.13",
-    "webpack": "4.16.1",
+    "webpack": "^4.42.1",
```
Comment (mickmister) [inline webapp/package.json:29]: Could you save the exact version here? Also the `>=` in the `lodash` dependency seems a little odd. Probably fine.

Comment (jfrerich) [reply webapp/package.json]: Ok. There's been quite a few testing commits, but I was able to get the exact version of webpack with `npm -E audit fix`.

**Comment by mickmister** [comment]
Suggesting we merge if the UI loads properly. It does not need an exhaustive regression test

**Comment by hmhealey** [comment]
@DHaussermann Do you have an estimate for when you'll be able to get to this? We're wanting to update the version of the NPS plugin in 5.23

**Comment by DHaussermann** [comment]
@hmhealey I can look at this this week. 

@ogi-m Let's connect to see if there's a subset of the release testing we can do to just regression test the core functionality.

**Comment by nevyangelova** [comment]
@nevyangelova , mentioning myself here so I can get notified when this gets merged as we need to roll out  https://mattermost.atlassian.net/browse/MM-23827 before code complete.

**Comment by DHaussermann** [comment]
@jfrerich I'm unable to adjust labels on this repo.
`QA Review` can be removed and `Reviews Completed` can be added

**Pull request merged**